pkg/sync/v6/data/Deployment.apps/openshift-azure-monitoring/metrics-bridge.yaml

```diff
@@ -44,6 +44,15 @@ spec:
         - /config/config.yaml
         image: '*** GENERATED ***'
         name: metricsbridge
+        ports:
+        - containerPort: 8080
+          name: http
+        readinessProbe:
+          httpGet:
+            path: healthz/ready
```

Comment:
Do we not need an initial `/` on the path?

Reply:
Not necessarily. This works without the initial slash. I'll add it however and we can update the other places in the codebase where its used in a follow-up.